### PR TITLE
ci: update badge whenever new interop dashboard is ready

### PR DESCRIPTION
**Pull request by galargh**
TODO: test

### CODE REVIEW

**Comment by galargh** [comment]
@laurentsenta We'll be able to use something like this to publish dashboard - https://github.com/libp2p/test-plans/issues/55

File: .github/workflows/update-badge.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          job: run-libp2p-test-plans
+      - uses: actions/checkout@v3
+      # https://github.com/orgs/community/discussions/26560
+      - run: |
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+      - run: |
+          in='\[\!\[Interop Dashboard\]\(https\:\/\/github\.com\/libp2p\/test\-plans\/actions\/workflows\/ping\-interop\-dashboard\.yml\/badge\.svg\?branch\=master\)\]\(.*\)'
+          out="[![Interop Dashboard](https://github.com/libp2p/test-plans/actions/workflows/ping-interop-dashboard.yml/badge.svg?branch=master)](${env.GITHUB_JOB_SUMMARY_URL})"
+          sed -i '' '#${in}#{out}#' README.md
```
Comment (laurentsenta) [inline .github/workflows/update-badge.yml:35]: I think we miss things here like using `${{ env.GITHUB ... }}` for github substitution, using double quote to allow variable expansion, etc.

I reused your code and updated it with: https://github.com/libp2p/test-plans/pull/88/commits/55b9be6e4489d6ef5589976836c1bbbd4546efa2

**Comment by laurentsenta** [comment]
Thanks for sharing @galargh !
We can close this PR, I created the complete feature that reuses this work in https://github.com/libp2p/test-plans/pull/88

I haven't retrieved a valid `job_summary_url` successfully with the action yet. Could you take a look there?

**Comment by p-shahi** [comment]
closing in favor of #88

**Pull request closed without merging**